strabo/static/js/upload_ips.js

```diff
@@ -1,50 +1,124 @@
-var layer_message;
-var icon_message;
-var map_message;
-var title_message;
+var InputField = function(message_identifier,is_valid_cond){
```

Comment:
So, you are using this instead of the form error we had originally in the HTML? Why did you change it?